README.md

```diff
@@ -1,12 +1,13 @@
-<div id="top"></div>
+![](https://img.shields.io/badge/Compatible%20with-Camunda%20Platform%207-blue)
+![](https://img.shields.io/badge/Compatible%20with-Camunda%20Platform%208-blue)
+![](https://img.shields.io/bower/l/bootstrap)
+![](https://img.shields.io/github/issues/FlowSquad/miranum)
 
-<!-- PROJECT LOGO -->
-<br />
 <div align="center">
     <a href="#">
-        <img src="/images/logo.png" alt="Logo" height="50">
+        <img src="/images/logo_blau.png" alt="Logo" height="50">
```

Comment:
I'd use the same image as in miranum-ide.
There's no need to change that here. I've added it in the rename PR already.

Reply:
Jup, that's right. The miranum-ide readme changes came past this PR. Due to this there they are not coordinated that well. 
Let's change this when merged your PR then 👍